src/network-handler.js

```diff
@@ -0,0 +1,28 @@
+var os = require('os');
+
+module.exports = function getVpnAddress() {
+  const ifaces = os.networkInterfaces();
+
+  // convert iface object into list
+  const ifaceList = Object.keys(ifaces).map((ifaceName) => {
+    const test = ifaces[ifaceName].map((iface) => {
+      iface.name = ifaceName;
+      return iface;
+    });
+    return test;
+  });
+
+  // flatten list
+  const flatIfaceList = ifaceList.flat();
```

Comment:
I get: `UnhandledPromiseRejectionWarning: TypeError: ifaceList.flat is not a function`

Do you know why?